ocaml/xapi/export_raw_vdi.ml

```diff
@@ -53,6 +53,12 @@ let localhost_handler rpc session_id vdi (req: Http.Request.t) (s: Unix.file_des
              Http_svr.headers s headers;
              match format with
              | Raw | Vhd ->
+               if format = Vhd then begin
+                 let size = Db.VDI.get_virtual_size ~__context ~self:vdi in
+                 if size > Constants.max_vhd_size
```

Comment:
Again - the code obtains a VDI size and compares it a gainst a VHD size.

Reply:
This is fine here, because we are exporting a VHD.